rero_ils/modules/ext.py

```diff
@@ -166,6 +166,29 @@ def __init__(self, app=None):
             ])
             app.jinja_loader = ils_loader
 
+            # patron information's
+            def patron_message():
+                """Get patron message."""
+                if not current_patrons:
+                    return
+                data = {
+                    'show_info': False,
+                    'data': {}
+                }
+                for patron in current_patrons:
+                    if (patron.is_blocked or patron.is_expired):
+                        data['show_info'] = True
+                    organisation = patron.organisation
+                    data['data'][organisation['code']] = {
+                        'name': organisation['name'],
+                        'blocked': {
+                            'is_blocked': patron.is_blocked,
+                            'message': patron.get_blocked_message(public=True)
+                        },
+                        'is_expired': patron.is_expired
+                    }
+                return data
```

Comment:
is it possible to move this into `patron.views.py` with a specific decorator ? Adding this into `ext.py` it's a little bit disturbing/annoying for me.